README.md

```diff
@@ -43,20 +43,20 @@ release (or version) of the project."
 
 ## Installation
 
-    gem install github_changelog_generator
+    # gem install github_changelog_generator
```

Comment:
Here, I would like _not_ to say `$` nor `#`.

The way people install Ruby varies wildly. Their best way to use it is about how they installed it.

Reply:
I would say `#`. People who are unfamiliar with ruby have it installed via their package manager. People how are familiar with ruby, know that they dont need to run it as root.

Maybe `$ sudo` would be better here.

Reply:
Some people take the long way around and `$ [sudo] gem install ...`

Reply:
Thats a good compromise. Changed it.